packages/jest-dom-mocks/src/intersection-observer.ts

```diff
@@ -44,6 +46,9 @@ export default class IntersectionObserverMock {
     const setObservers = (setter: (observers: Observer[]) => Observer[]) =>
       (this.observers = setter(this.observers));
 
+    (global as any).IntersectionObserverEntry = () => {};
+    (global as any).IntersectionObserverEntry.prototype.intersectionRatio = 0;
```

Comment:
This is usually a getter right? Might be better to match the property descriptor a little better.